playbooks/aws/README.md

```diff
@@ -48,13 +49,24 @@ provision:
   clusterid: "{{ clusterid }}"
   region: "{{ region }}"
 
+  # configuration for different components
+  create_vpc: True
+  create_ssh_keys: True
+  create_s3: True
+  create_iam_cert: True
+  create_security_groups: True
+
   build:
+    ami_name: "openshift-gi-"
     base_image: ami-bdd5d6ab # base image for AMI to build from
```

Comment:
Let's remove this ami from the docs and repo if it's not accessible by users.

Reply:
We can remove this one but the user will still need to specify an ami for this `base_image`.

Reply:
Removed.